aws-cli/bash-linux/dynamodb/dynamodb_operations.sh

```diff
@@ -17,6 +17,7 @@
 #
 ###############################################################################
 
+# shellcheck disable=SC1091
```

Comment:
Is disabling this check correct? I'm curious why shellcheck can't find `awsdocs_general.sh`, it's right there? (Same for other SC1091 checks)

Reply:
My guess is that shellcheck is running from a different directory.

Reply:
I run the tests from the test directory.

Reply:
Interesting - can you use the `# shellcheck source=` syntax?

Reply:
I fixed it and removed all those suppress statements